Library/Homebrew/cmd/audit.rb

```diff
@@ -182,6 +182,10 @@ def audit_file
       problem "'__END__' was found, but 'DATA' is not used"
     end
 
+    if text =~ /inreplace .* do .*$\n.*\.gsub!?.*$\n\ *end/m
```

Comment:
Because `/…/m` causes `.` to also match `\n`, this produces many false positives for valid use cases. (Also, having `$\n` inside the regular expression looks weird.) A few examples: `brew audit afl-fuzz akka ansible bsdmake`.